packages/sui-cz/PrompterManager.js

```diff
@@ -0,0 +1,188 @@
+   * @param {string} commitString
+   */
+  static doCommit (commitString) {
+    ErrorCommitSaver.initErrorListener(commitString)
+    this.commit(commitString)
+    ErrorCommitSaver.discardOldCommit()
+  }
+
+  /**
+   * The get commmit steps method will return an array of objects that corresponds to the different steps of our commit promptet.
+   * [Step1, ...2, ...3, ...4, ...5, ...6, ...7]
+   * @returns {[Object,Object,Object,Object,Object,Object,Object]}
+   */
+  static getCommitSteps () {
+    return [
+      {
+        type: 'list',
+        name: 'type',
+        message: "Select the type of change that you're committing:",
+        choices: getCommitTypesMapped()
+      },
+      {
+        type: 'list',
+        name: 'scope',
+        message: '\nDenote the SCOPE of this change:',
+        choices: answers => {
+          return Promise.all(
+            scopes.map(pkg => this.checkIfHasChangedFiles(path.join(packagesDir, pkg.name)).then(hasFiles => hasFiles && pkg))
+          )
+            .then(result => result.filter(Boolean))
+            .then(result => {
+              if (typesWithOtherScopes.indexOf(answers.type) > -1) {
+                return result.concat(otherScopes)
+              }
+              return result
+            })
+        }
+      },
+      {
+        type: 'input',
+        name: 'subject',
+        message: 'Write a SHORT, IMPERATIVE tense description of the change:\n',
+        validate: value => !!value,
+        filter: value => (value.charAt(0).toLowerCase() + value.slice(1))
+      },
+      {
+        type: 'input',
+        name: 'body',
+        message: 'Provide a LONGER description of the change (optional). Use "|" to break new line:\n'
+      },
+      {
+        type: 'input',
+        name: 'breaking',
+        message: 'List any BREAKING CHANGES (optional):\n',
+        when: answers => allowedBreakingChanges.indexOf(answers.type.toLowerCase()) > -1
+      },
+      {
+        type: 'input',
+        name: 'footer',
+        message: 'List any ISSUES CLOSED by this change (optional). E.g.: #31, #34:\n'
+      },
+      {
+        type: 'expand',
+        name: 'confirmCommit',
+        choices: [
+          {key: 'y', name: 'Yes', value: 'yes'},
+          {key: 'n', name: 'Abort commit', value: 'no'},
+          {key: 'e', name: 'Edit message', value: 'edit'}
+        ],
+        message: function (answers) {
+          const SEP = '###--------------------------------------------------------###'
+          console.log(`${SEP} \n ${buildCommit(answers)} \n ${SEP} \n`)
+          return 'Are you sure you want to proceed with the commit above?'
+        }
+      }
+    ]
+  }
+
+  static startRecoverOldCommitFlow (cz, commit, commitString) {
+    const HR = '\n ******************************************************************\n'
+    console.log(colors.yellow(`${HR} ${commitString} ${HR} -> We founded that you haved problems with the last commit and we saved the message for you 😇.`))
```

Comment:
typos

`We found that you had problems with the last commit and we saved the message for you`